proto/rill/runtime/v1/api.proto

```diff
@@ -251,6 +251,8 @@ message Connector {
   string name = 2;
   // Config for the connector
   map<string, string> config = 3;
+  // config_from_variables stores a copy of configs whose value is a variable. The key is the property and value is the variable. This is only set for configs obtained from `connector.yaml`
+  map<string, string> config_from_variables = 4;
```

Comment:
This is not very great that we are storing a field for internal usage in API object. May be we can add another struct to store data in DB in future and remove this field from API struct.

Reply:
Agree. Might make sense to leverage the new `ConnectorSpec` internally instead.

Reply:
I can submit a follow up PR for this change. 
Also we need to introduce `name` field in `ConnectorSpec` to make it compatible with `connector` which will be somewhat unintuitive. Ideally spec is same for all names.